### DISCUSSION TITLE
option for ignoring certain file types

### DISCUSSION THREAD

**Discussion by notwidow** [helix-editor/helix#3891] [Q&A]
Hi, I have an issue for binary files. for file picker max-depth = 2 set. 
so file-picker shows binary files as well. is there an option to ignore certain types of files in file-picker...?
**Steps to reproduce**
set the max-depth to 1 or 2
for file picker and hit ** space ** followed by ** f ** ....
thanks...
![b0](https://user-images.githubusercontent.com/112823418/190916690-7345c321-32c6-497b-92d2-cf40b1827998.png)
![b1](https://user-images.githubusercontent.com/112823418/190916694-470a54ee-7f19-4be3-ae79-5006ac1f0d25.png)

**Comment by David-Else** [comment]
You can add those to a `.gitignore` file, but I agree it would be good to hide known binaries by default.

**Comment by the-mikedavis** [comment]
You can use a `.ignore` file if the files are outside of a git directory

**Reply by notwidow** [reply]
> You can use a `.ignore` file if the files are outside of a git directory
well there are many files not one or two ... and they are everywhere in system so the solution will be to have an option for ignoring filetypes.....

**Reply by the-mikedavis** [reply]
The `.ignore` file can ignore extensions. For example:

```
*.kbx
*~
*.gpg
*.x86_64-pc-linux-gnu
```

**Comment by notwidow** [comment]
this is my 2nd issue you closed without providing solution. this is bad ...

**Reply by the-mikedavis** [reply]
I converted both to discussions. The GitHub UI makes the original issue look closed but it's just a different format.

In general, please use discussions or the Matrix room for questions instead of issues.
